interpreter/cling/lib/Interpreter/Interpreter.cpp

```diff
@@ -220,6 +220,20 @@ namespace cling {
     if (!m_IncrParser->isValid(false))
       return;
 
+    // Load any requested plugins.
+    getCI()->LoadRequestedPlugins();
+
+    // Honor -mllvm. This should happen AFTER plugins have been loaded!
```

Comment:
```suggestion
    // Honor set of `-mllvm` options. This should happen AFTER plugins have been loaded!
```

Reply:
This was merged principally to test whether the new CI is properly picking up the current master. (and because it is a mild improvement)